deploy/examples/cluster.yaml

```diff
@@ -281,6 +281,21 @@ spec:
     # No values or 0 means that the operator will wait until the placement groups are healthy before unblocking the next drain.
     pgHealthCheckTimeout: 0
 
+  # csi defines CSI Driver settings applied per cluster.
+  csi:
+    readAffinity:
+      # Set enabled to true to enable read affinity.
+      # For more details see this doc:
+      # https://rook.io/docs/rook/latest/Storage-Configuration/Ceph-CSI/ceph-csi-drivers/#enable-read-affinity-for-rbd-volumes
+      enabled: false
```

Comment:
Remind me why this isn't enabled by default? If there is some concern to enable it, we should tell the user here.

Reply:
> Remind me why this isn't enabled by default? If there is some concern to enable it, we should tell the user here.

There is increased OSD memory usage since multiple OSDs in different zones will be serving the same piece of data for clients in that zone instead of the data coming from the same primary OSD.

Reply:
Ok, how about adding a comment to the setting here? Even if we don't have specific numbers, something like this seems helpful: 
```
# Enabling the read affinity will cause the OSDs to consume some extra memory.
```